WorkloadManagementSystem/Agent/JobCleaningAgent.py

```diff
@@ -5,7 +5,15 @@
 """ The Job Cleaning Agent controls removing jobs from the WMS in the end of their life cycle.
 
     This agent will take care of removing user jobs,
-    while production jobs should be removed by the TransformationCleaningAgent.
+    while production jobs should be removed by the :mod:`~DIRAC.TransformationSystem.Agent.TransformationCleaningAgent`.
```

Comment:
Actually, this is not exactly correct: the JobCleaningAgent really removes Jobs from the DB, while the TransformationCleaningAgent will change the status of the jobs as "Deleted"

Reply:
`should be removed by the` --> `should be removed through the` ?
BTW: https://github.com/DIRACGrid/DIRAC/commit/a513fbab085a85d93480aadf9d8db9f9ac3eee1e :smiling_imp:

Reply:
Hey, I was too young at the time!!!!

 Thanks for correcting it.